lnst/Recipes/ENRT/VirtualBridgeVlanInGuestMirroredRecipe.py

```diff
@@ -60,18 +64,16 @@ def test_wide_configuration(self):
         guest1.vlan0 = VlanDevice(realdev=guest1.eth0, vlan_id=self.params.vlan_id)
         guest2.vlan0 = VlanDevice(realdev=guest2.eth0, vlan_id=self.params.vlan_id)
 
-        configuration = super().test_wide_configuration()
-        configuration.test_wide_devices = [guest1.vlan0, guest2.vlan0,
-            host1.br0, host2.br0]
+        config = super().test_wide_configuration()
 
         ipv4_addr = interface_addresses(self.params.net_ipv4)
         ipv6_addr = interface_addresses(self.params.net_ipv6, default_start="fc00:0:0:1::3/64")
-        host1.br0.ip_add(next(ipv4_addr))
-        host2.br0.ip_add(next(ipv4_addr))
+        config.configure_and_register_ip(host1.br0, next(ipv4_addr))
+        config.configure_and_register_ip(host2.br0, next(ipv6_addr))
```

Comment:
I think this is a typo. Originally, the test configures ipv4 address on host1.br0 and host2.br0 (the other host). Now you configure ipv4 address on host1 and ipv6 address on other host.

Reply:
A very good catch. Thanks, fixed.